wpimath/src/main/native/include/frc/util/PIDConstants.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+namespace frc {
+
+/** A class to configure a PIDController with */
+struct PIDConstants {
+ public:
+  /** P */
+  double Kp = 0.0;
+  /** I */
+  double Ki = 0.0;
+  /** D */
+  double Kd = 0.0;
```

Comment:
Is there a particular reason the java counterpart to this marks these as final but the C++ version does not mark them const? There is no real reason to check these values for their domain at construction time if they are not actually constants.

Reply:
Nope I forgot to change them when I made the Java ones final